Assets/BossRoom/Scripts/Client/Game/Entity/ClientProjectileVisualization.cs

```diff
@@ -31,6 +31,14 @@ public override void NetworkStart()
             m_NetState.HitEnemyEvent += OnEnemyHit;
         }
 
+        private void OnDestroy()
```

Comment:
Question: If I implement a subclass of this `ClientProjectilkeVisualization` that has a requirement to implement some additional custom logic OnDestroy, how is this `private` modifier allowing me to do that if I also want to run the parent's class implementation? 

The same question applies everywhere where you just added OnDestroy functionality.

Reply:
I hadn't really thought of those classes as likely candidates for specialization, but yeah, I agree, if they were specialized and had specialized destroy logic, you'd need to refactor the parent class. You'd prefer `protected` I assume? I'll happily go change them.

Reply:
actually, it occurs to me perhaps you meant "why have an access modifier at all", since Unity message functions commonly do not. I will try that--at least it will be consistent with other Start and Update methods in the code base.